index.html

```diff
@@ -280,17 +280,16 @@ <h1>
 
     <p>
 This specification utilizes registries to enable interoperable definitions of
-identifiers used by DID implementations, including
-<a>DID document</a> member names defined by this and other specifications,
-<a>DID methods</a>,
+<a>DID document</a> property names defined by this and other specifications,
+<a>DID method</a> names, DID parameter names,
 and cryptographic algorithm identifiers.
 Some of these registries are operated by the W3C DID Working group
 and others are operated by other standards organizations, including the IETF.
     </p>
 
     <p>
 This design eliminates dependence on centralized registries
-for key management, which is the
+for identifier and key management, which is the
```

Comment:
Adding "identifier and" is incorrect here because we are relying on working group registries for managing identifiers, including property names, algorithms names, and parameter names.  Please delete "identifier and".  I'm fine with the rest of the changes.

Reply:
The "identifier" part of this is referring to the identifiers used in the Verifiable Data Registry (e.g., the DLT network) .... not the identifiers used in the "DID Specification Registries". I believe the change @peacekeeper suggested is correct.

Reply:
As worded, it doesn't say that.  It makes it sound like registries aren't used for any identifiers, which is clearly false.

Please change the "identifier and" added to something more specific such as "identifier in the Verifiable Data Registry and" so that it's not misleading or simply delete it, as the PR did.

Reply:
Removed the word "registry" in 200cff00c827a58de4e4897f644838221699d0bd. It now reads:

> This design eliminates dependence on centralized identifier and key management systems, which is the standard in hierarchical PKI (public key infrastructure). 

Is that better?

Also note that this section is in the process of getting heavily rewritten, so I expect large swaths of text to not survive the next week... just as a heads-up.

Reply:
The working group decided to take a dependency on centralized registries.  We should own that and not try to imply that we didn't.  The current text is still misleading.  How about this proposed wording?

> This design eliminates dependence on centralized identifier and key management systems, other than the registries established by and used by this specification.  Note that centralized identifier and key management is used by existing hierarchical PKI (public key infrastructure) systems.